tests/acceptance/features/apiGraphUserGroup/getUser.feature

```diff
@@ -1594,3 +1599,73 @@ Feature: get users
       }
     }
     """
+
+  @env-config
+  Scenario Outline: user searches user and see users email if ShowUserEmailInResults config is enabled
+    Given the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "true"
+    And the administrator has assigned the role "<user-role>" to user "Alice" using the Graph API
+    When user "Alice" searches for user "Brian" using Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+    """
+    {
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "minItems": 1,
+          "maxItems": 1,
+          "items": {
+            "type": "object",
+            "required": [
+              "displayName",
+              "id",
+              "userType",
+              "mail"
+            ],
+            "properties": {
+              "displayName": {
+                "type": "string",
+                "enum": ["Brian Murphy"]
+              },
+              "id": {
+                "type": "string",
+                "pattern": "^%user_id_pattern%$"
+              },
+              "userType": {
+                "type": "string",
+                "enum": ["Member"]
+              },
+              "mail": {
+                "type": "string",
+                "enum": ["brian@example.org"]
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+     Examples:
+      | user-role   |
+      | Admin       |
+      | Space Admin |
+      | User        |
+      | User Light  |
+
+
+  @env-config
+  Scenario Outline: non-admin searches user and don't see user email if ShowUserEmailInResults config is disabled
+    Given the config "OCIS_SHOW_USER_EMAIL_IN_RESULTS" has been set to "false"
+    And the administrator has assigned the role "<user-role>" to user "Alice" using the Graph API
+    When user "Alice" searches for user "Brian" using Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the search response should not contain user email
```

Comment:
IMO, we can just add step `And the JSON data of the search response should not contain user email` to the following existing scenarios:
https://github.com/owncloud/ocis/blob/1d388a00a922e11ed5e8289e34295d9b09f3f5d5/tests/acceptance/features/apiGraphUserGroup/getUser.feature#L1334-L1337
https://github.com/owncloud/ocis/blob/1d388a00a922e11ed5e8289e34295d9b09f3f5d5/tests/acceptance/features/apiGraphUserGroup/getUser.feature#L1437-L1440

Reply:
already added